Pipfile.lock

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
The Pipfile.lock enables deterministic builds - just meaning that the **exact** same code will be utilized. We can remove this.